src/parser/druidsql.pegjs

```diff
@@ -568,25 +568,45 @@ SqlInParens = OpenParen leftSpacing:_? ex:Sql rightSpacing:_? CloseParen
   return ex.addParens(leftSpacing, rightSpacing);
 }
 
-SqlLiteral = lit:(Number / SingleQuotedString)
+SqlLiteral = lit:(Number / SingleQuotedString )
 {
   return new sql.SqlLiteral(lit);
 }
+/ BooleanLiteral
+/ NullLiteral
 / SqlInParens
 
-Number = n:$([0-9]+)
+NullLiteral = value: "NULL"i {
+    return new sql.SqlLiteral({value: value, stringValue: value, quotes:''});
+}
+
+BooleanLiteral = value: ("TRUE"i/"FALSE"i) {
+  return new sql.SqlLiteral({value: value, stringValue: value, quotes:''});
```

Comment:
Where is the test for this? Also wouldn't `value` be like `"TrUe"` ? It should be `true`

Reply:
As it is i can accept any capitalization of true, are you saying it should only accept lower case true/false?